fe/fe-core/src/main/java/com/starrocks/external/hive/HiveMetaClient.java

```diff
@@ -213,6 +223,67 @@ public HivePartition getPartition(String dbName, String tableName, List<String>
         }
     }
 
+    public HivePartition getHudiPartition(String dbName, String tableName, List<String> partitionValues) throws DdlException {
```

Comment:
how about add hive catalog and call hivemetaclient there?

Reply:
for reuse cache with hive here.

As the issue said, catalog related abstraction is not the goal of this PR, I will consider it in the following step.